_data/simple-icons.json

```
@@ -16792,6 +16792,11 @@
             "source": "https://venmo.com/about/brand",
             "guidelines": "https://venmo.com/about/brand"
         },
+	    {
+	        "title": "Vencord",
+	        "hex": "8E71D4",
```

```suggestion
	        "hex": "D3859B",
```
`#d3859b` seems like the Vencord theme primary accent.